static/src/js/app/vuex/tests/actions.spec.js

```diff
@@ -62,38 +104,51 @@ describe('Actions', () => {
         lang: 'lat',
         vocabularyListId: 1,
       };
-      //   const response = { data: { created: true } };
+      // payload sent to api function
+      const { lemmaId, vocabularyListId, ...apiPayload } = payload;
+      apiPayload.lemma_id = lemmaId;
+      apiPayload.vocabulary_list_id = vocabularyListId;
```

Comment:
I was a little puzzled by this at first, not because the test is unclear, but because I wasn't sure why we couldn't just use the same payload for both the action and API request. If I'm understanding correctly, it's because we need to rename the parameters from _camelCase_ to _underscore_ format since that's what the API endpoints expect. Not a big deal, but seems like unnecessary complexity we've introduced inadvertently. As we continue to address tech debt, we may want to just use camelCase on the frontend so that way we don't need to do this kind of transformation. The API endpoints could do what they want after they've received the parameters, of course.

Reply:
That's exactly it, those two keys just need to be renamed to use underscores when sending data to the API. Our js linter prefers camelCase variable names when possible, but on the django end underscores are used more often. Definitely something to make more consistent in the future.